### PR TITLE
Ab warning dialog

### PR DESCRIPTION
**Pull request by Cryotechnic**
This PR fixes an issue where the warning dialog that would show if a specific registry key was detected, would show every time the region was loaded. This is now limited to only when the user opens the Honkai: Star Rail settings page, and will appear every time it is loaded.

### CODE REVIEW

File: CollapseLauncher/XAMLs/MainApp/Pages/StarRailGameSettingsPage.xaml.cs [hunk 1]
```diff
@@ -78,8 +80,16 @@ private void LoadPage()
             GameSettingsApplyGrid.Translation = new System.Numerics.Vector3(0, 0, 64);
 
             InheritApplyTextColor = ApplyText.Foreground;
+            
+            // A/B Testing as of 2023-12-26 (HSR v1.6.0)
+            object? abValue = Registry.GetValue(@"HKEY_CURRENT_USER\Software\Cognosphere\Star Rail", _AbValueName, null);
```
Comment (bagusnl) [inline CollapseLauncher/XAMLs/MainApp/Pages/StarRailGameSettingsPage.xaml.cs:85]: Put this line under `#nullable enable` context and close the nullable at line 90

File: Hi3Helper.Core/Lang/en_US.json [hunk 2]
```diff
@@ -74,6 +74,8 @@
     "UnhandledSubtitle2": "Oops, it seems like you've been disconnected from the Internet~ Or is it something else?",
     "UnhandledTitle3": "Game Crashed",
     "UnhandledSubtitle3": "The game has crashed with error details below:",
+    "UnhandledTitle4": "Warning",
+    "UnhandledSubtitle4": "This isn't a major issue, but we thought we should let you know:\r\n\r\nDue to miHoYo A/B testing, Collapse does not support reading the following key: App_Settings_h2319593470.\r\n\r\nWe apologize for the inconvenience and thank you for your understanding.",
```
Comment (bagusnl) [inline Hi3Helper.Core/Lang/en_US.json:78]: There seems to be too much spacing between paragraphs, especially the last one. A single `\r\n` should do just fine.

**Review by bagusnl — CHANGES_REQUESTED** [review]
Overall code looks good, just needs a little attention in the subtitles spacing and nullable context

File: CollapseLauncher/XAMLs/MainApp/Pages/StarRailGameSettingsPage.xaml.cs (hunk 1 above)
Comment (Cryotechnic) [reply CollapseLauncher/XAMLs/MainApp/Pages/StarRailGameSettingsPage.xaml.cs:85]: Fixed in 9c5c173

File: Hi3Helper.Core/Lang/en_US.json (hunk 2 above)
Comment (Cryotechnic) [reply Hi3Helper.Core/Lang/en_US.json]: See above

**Review by bagusnl — APPROVED** [review]
All good, approb

**Comment by Cryotechnic** [comment]
> All good, approb

image

**Pull request merged**